slides/workshop_rap_for_managers_basics.qmd

```diff
@@ -17,27 +17,35 @@ format:
 
 # Why am I here?
 
-This session is designed to cover some of the basic principles of RAP from the perspective of a manager, and to give you a chance to ask any questions that you might have.
+This session is designed to cover some of the basic principles of RAP from the perspective of a manager, to outline the level of technical skills (R, SQL, Git) that we expect, and to give you a chance to ask any questions that you might have.
+
+Coding and technical skills form a core part of our work as analysts in DfE. To continue producing high quality statistics, teams need to maintain consistent levels of capability at all levels, and so should be looking to build these skills at every opportunity. 
 
 <br>
 
-Our main aims are to:
+In this workshop, our main aims are to:
```

Comment:
Consider adding a reference to the second workshop that will be coming and how it builds on this content somewhere.